flipperdevices/flipperzero-good-faps
# New nfc_magic options for gen4 cards

# Credits

- UI / Scenes - by @xMasterX
- Poller functions, minor UI fixes - by @Leptopt1los

# What's new

For Gen4 cards:

- Auth with password option moved into new submenu "Gen4 actions"
- New function: Get card revision
- New function: Get card config (shows only when debug ON)
- New function: Set default config: for 06 A0 cards "revival". This revision of the cards may "fall into a coma" when interacting with it. In this state, she does not respond to any commands. This problem is solved by writing a standard config to the card

# Verification 

- Get revision for gen4 card
- Get revision for non-gen4 card
- Get card config for gen4 card
- Get card config for non-gen4 card
- Set default config for gen4 card
- Set default config for non-gen4 card

All actions with the gen4 card must be checked for:

- on the card 06 A0 revision in working condition
- on the card 06 A0 revision in non-working condition (card didn't detect by check magic tag)
- on the card non 06 A0 revision

After "set default config" success card must be detected via "check magic tag" option

# Checklist (For Reviewer)
